src/main.py

```diff
@@ -1,18 +1,21 @@
 import os
-import requests
 import pandas as pd
+from dune_client.client import DuneClient
+from dune_client.query import QueryBase
 from sqlalchemy import create_engine
 from dotenv import load_dotenv
 
 from src.types import DUNE_TO_PG
 
 load_dotenv()
-DUNE_API_KEY = os.getenv("DUNE_API_KEY")
-DB_URL = os.getenv("DB_URL")
-TABLE_NAME = "dune_data"
+DUNE_API_KEY = os.environ.get("DUNE_API_KEY")
+DB_URL = os.environ.get("DB_URL")
 
-API_URL = "https://api.dune.com/api/v1/query/4132129/results"
-HEADERS = {"x-dune-api-key": DUNE_API_KEY}
+# TODO(bh2smith): parse config file for most of the following stuff
+QUERY_ID = int(os.environ.get("QUERY_ID"))
+POLL_FREQUENCY = int(os.environ.get("POLL_FREQUENCY"))
+QUERY_ENGINE = os.environ.get("QUERY_ENGINE")
+TABLE_NAME = f"dune_data_{QUERY_ID}"
```

Comment:
nitpick: need to make sure that pandas' `to_sql` doesn't create funky SQL that could end up running injections on the server.

Reply:
Yea this makes sense to consider, but remember that the user of this project will be in control of the input stream (i.e. whatever the query results may be). Its up to them to protect themselves from such a thing. Whose to say they don't want to include SQL injection... This may not, necessarily be our concern.

However, we may eventually want to substitute the pandas `to_sql` out for something we have more control over... Not sure though, because it works so well for now.

Reply:
I see your point too, anti-sqli from owner-supplied vectors might be too much to consider right now.